src/HCSR04.h

```diff
@@ -13,8 +13,10 @@ class UltraSonicDistanceSensor {
     /**
      * @param triggerPin  Digital pin that is used for controlling sensor (output).
      * @param echoPin  Digital pin that is used to get information from sensor (input).
+     * @param maxDistanceCm  Maximum distance sensor can measure, defaults to 4m for HC-SR04.
+     * @param timeoutMicroSec  Set absolute measurement timeout.
```

Comment:
We could mention here that this is useful if you need to be sure that measurement will not take longer than some specific time. But, this also means that we are limiting max distance it can measure.